src/select/Select.jsx

```diff
@@ -130,6 +130,9 @@ class Select extends Component {
     }
 
     if (state.visible != this.state.visible) {
+      if (this.props.onVisibleChange) {
```

Comment:
In Docs for the Select component in part "Select Events" there is an onVisibleChange event,
but it doesn't work:
![image](https://user-images.githubusercontent.com/9878222/29560059-43a8b328-874a-11e7-9ed8-62a9f37f7c68.png)
My code for example:

```
<Select onVisibleChange={ () => console.log("does not appear :(") }>
  {options.map(option => (
    <Select.Option
      key={ option.value }
      value={ option.value }
      label={ option.label }
    />
  ))}
</Select>
```